### PR TITLE
Update readme

### LINKED ISSUE #409: README updates post 1/9 meeting
- [x] mention bash_profile for creds for mac
- [x] distinguish in readme about mandatory vs non-mandatory instructions
- [x] ask someone to add you to the cypress dashboard (optional)
- [x] cypress how to run test instructions
- [x] add example.json .gitignore 
- [x] don't put your secret inside the github repository
- [x] run yarn if you have an issue (will install deps)
- [x] link to git basics
- [x] add advice to use lint.sh instead of clang-format etc/Replace clang-format/eslint with lint.sh.
- [x] add ogr2ogr install/run suggestions
- [x] earth engine whitelist yourself?

### PR DESCRIPTION
**Pull request by juliexxia**
closes #409

also add everything under cypress/fixtures to .gitignore

Easy link to looking at edited page in nice md: https://github.com/givedirectly/Google-Partnership/tree/update-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,70 @@
...
 
-* Start the local web server by running
-`yarn run ws --directory docs --port 8080`.
+* Set up auto-lint: `echo ./pre-push-hook.sh > .git/hooks/pre-push`
 
-* Set `export GOOGLE_APPLICATION_CREDENTIALS=/path/to/secret.json` in your
-shell, most likely in your `~/.bashrc`. Get the service account .json file,
-either from a collaborator or by logging into the [Service Accounts page](
+* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and format in
+place. Clang-format will run automatically on commit and eslint will run automatically on push so
```
Comment (janakdr) [inline README.md]: I don't think this is quite right. clang-format doesn't run on commit, but running it then requires you to recommit. Maybe rephrase?

File: README.md [hunk 2]
```diff
@@ -1,58 +1,70 @@
...
 
 * Test changes locally.
 
     ```shell
-    yarn run cypress run # --browser chromium ## (only needed on Linux) 
+    yarn run cypress run # --browser chrome ## (only needed on Linux) 
     ```
 
-* Travis CI runs on each push to Github, unless the commit message has
+* *[Optional]* Travis CI runs on each push to Github, unless the commit message has
```
Comment (janakdr) [inline README.md]: this is now stale, only runs on pushes to PRs.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,70 @@
...
 '[skip ci]' inside it. You can trigger a run manually from the [Travis main
 page](https://travis-ci.com/givedirectly/Google-Partnership). Runs are recorded
 to the [Cypress dashboard](https://www.cypress.io/dashboard/).
 
-* Save and share your personal test results in a web interface using
-[Cypress dashboard](https://www.cypress.io/dashboard/). The dashboard also
-provides easy access to output files, footage of the test being run, etc. To find
-the record key for this project go to the Google-Partnership project at
-https://dashboard.cypress.io in the "Runs" tab under "Settings".
+* *[Optional]* Save and share your personal test results in a web interface using
```
Comment (janakdr) [inline README.md]: I think maybe we can drop this? No real reason to put non-Travis results in the dashboard, right?

File: README.md [hunk 4]
```diff
@@ -61,21 +73,29 @@ https://dashboard.cypress.io in the "Runs" tab under "Settings".
...
-
-    clang-format -i --style=Google $SOURCE_FILES
-    yarn run eslint $SOURCE_FILES
+    brew install gdal
+    ogr2ogr -f "ESRI Shapefile" destination_data.shp "source-data.json"
     ```
 
+## Helpful Links
+* [git documentation](https://git-scm.com/docs)
+* [cypress test running documentation](https://docs.cypress.io/guides/guides/command-line.html#cypress-run)
```
Comment (janakdr) [inline README.md]: nit: "test-running"

File: README.md (hunk 3 above)
Comment (juliexxia) [reply README.md]: Should still link to the dashboard somewhere since it's nice. Let me know what you think of revised.

File: README.md (hunk 1 above)
Comment (juliexxia) [reply README.md]: Updated. Let me know what you think

File: README.md [hunk 5]
```diff
@@ -30,9 +30,9 @@ and visit the page at `localhost:8080` (we recommend in Chrome).
 
 * Set up auto-lint: `echo ./pre-push-hook.sh > .git/hooks/pre-push`
 
-* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and format in
-place. Clang-format will run automatically on commit and eslint will run automatically on push so
-this just saves you the extra commits. 
+* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and (with `--fix`) format in
+place. Clang-format and eslint will run automatically on `git push` (without `--fix`) so this just
```
Comment (janakdr) [inline README.md]: maybe "Clang-format checking", since clang-format doesn't format in place on pushes
